src/main/java/org/spectra/cluster/tools/CliOptions.java

```diff
@@ -0,0 +1,231 @@
+ * Time: 11:35 AM
+
+ */
+public class CliOptions {
+    public enum OPTIONS {
+
+        CONFIG_FILE("config", "c"),
+        OUTPUT_PATH("output.path", "o"),
+
+        INPUT_FILES("input.files", "i"),
+
+        PRECURSOR_TOLERANCE("precursor.tolerance", "p"),
+        FRAGMENT_TOLERANCE("fragment.tolerance", "f"),
+
+        MAJOR_PEAK_JOBS("major.peak.jobs", "m"),
+
+        START_THRESHOLD("threshold.start", "s"),
+        END_THRESHOLD("threshold.end", "e"),
+
+        ROUNDS("rounds", "r"),
+
+        BINARY_TMP_DIR("binary.directory", "b"),
+        KEEP_BINARY_FILE("keep.binary.files", "k"),
+
+        REUSE_BINARY_FILES("reuse.binary.files", "u"),
+        REMOVE_REPORTER_PEAKS("remove.reporters", "rr"),
+
+        FAST_MODE("fast_mode", "fm"),
+        FILTER("filter", "ft"),
+        HELP("help", "h"),
+        VERBOSE("verbose", "v"),
+
+        // Advanced options
+        ADVANCED_MIN_NUMBER_COMPARISONS("x.min.comparisons", "mc"),
+        ADVANCED_NUMBER_PREFILTERED_PEAKS("xn.prefiltered.peaks", "pp"),
+        ADVANCED_LEARN_CDF("x.learn.cdf", "lcdf"),
+        ADVANCED_LOAD_CDF_FILE("x.load.cdf", "lcdf"),
+        ADVANCED_DISABLE_MGF_COMMENTS("disable.mgf.comments", "dmc");
+
+        private String value;
+        private String longValue;
+
+        OPTIONS(String logValue, String value) {
+            this.value = value;
+            this.longValue = logValue;
+        }
+
+        public String getValue() {
+            return value;
+        }
+
+        @Override
+        public String toString() {
+            return value;
+        }
+
+        public String getLongValue() {
+            return longValue;
+        }
+    }
+
+    private static final Options options = new Options();
+
+    static {
+
+        Option config = OptionBuilder
+                .hasArg()
+                .withDescription("Input file with all the configuration parameters")
+                .withLongOpt(OPTIONS.CONFIG_FILE.getLongValue())
+                .create(OPTIONS.CONFIG_FILE.getValue());
+        options.addOption(config);
+
+        Option inputFiles = OptionBuilder
+                .hasArgs(5)
+                .withDescription("Input files to be analyzed. If more than one file is provided it should be with space ")
+                .withLongOpt(OPTIONS.INPUT_FILES.getLongValue())
+                .create(OPTIONS.INPUT_FILES.getValue());
+        options.addOption(inputFiles);
+
+        Option fragmentTolerance = OptionBuilder
+                .hasArg()
+                .withDescription("Fragment ion tolerance in m/z to use for fragment peak matching")
+                .withLongOpt(OPTIONS.FRAGMENT_TOLERANCE.getLongValue())
+                .create(OPTIONS.FRAGMENT_TOLERANCE.getValue());
+        options.addOption(fragmentTolerance);
+
+        Option precursorTolerance = OptionBuilder
+                .hasArg()
+                .withDescription("Precursor tolerance (clustering window size) in m/z used during matching.")
```

Comment:
We need to be careful with this description. If we define it as the tolerance, I'd personally expect the windows size to be tolerance * 2 (that's how it was used in the original version). Personally, I'd stick with this approach and remove the `window size` from the description.

```suggestion
                .withDescription("Precursor tolerance in m/z used during matching.")
```